src/schema/sale_artworks.js

```diff
@@ -30,11 +30,19 @@ export default {
     _root,
     options,
     _request,
-    { rootValue: { saleArtworksFilterLoader } }
+    { rootValue: { saleArtworksFilterLoader, saleArtworksAllLoader } }
   ) => {
     const relayOptions = { ...DEFAULTS, ...options }
     const params = parseRelayOptions(relayOptions)
-    const response = await saleArtworksFilterLoader(params)
+    let fetchLoader = saleArtworksFilterLoader
+
+    if (options.live_sale) {
+      fetchLoader = saleArtworksAllLoader
+    }
```

Comment:
@ashkan18  - And this is where I'm determining which loader to use, based on the query parameter being passed in. 

My question about the pagination error

> "https://stagingapi.artsy.net/api/v1/sale_artworks?include_artworks_by_followed_artists=true&live_sale=true&offset=0&page=1&size=10 - {\"type\":\"param_error\",\"message\":\"Invalid parameters.\",\"detail\":{\"page, offset\":[\"are mutually exclusive\"],\"artwork_id, artworks_by_followed_artists\":[\"are missing, exactly one parameter must be provided\"],\"artworks_by_followed_artists, live_sale\":[\"provide all or none of parameters\"]}}"

centers around how the old query doesn't throw with default settings, yet the new one being referenced above (in the conditional) does; makes me think there's a subtle difference in how the two endpoints handle pagination (or a mistake I've introduced somewhere)

Reply:
i think the error above is because of missing `artworks_by_follow_artists` param. 
```
"artwork_id, artworks_by_followed_artists":["are missing, exactly one parameter must be provided"],"artworks_by_followed_artists, live_sale":["provide all or none of parameters"]
```
I wonder if thats what the real error is, looking at pagination looks like `filter` endpoint does things slightly different than how we do it every where else where we add params using `use pagination`
https://github.com/artsy/gravity/blob/master/app/api/v1/sales_artworks_endpoint.rb#L24 which ends up adding these params
https://github.com/artsy/gravity/blob/master/app/api/util/pagination_parameters.rb#L6
notice how we have
```
mutually_exclusive :page, :offset, :cursor
```

Reply:
Yea, we've had these issues in Metaphysics <-> Gravity for a while. Some endpoints in Gravity _do_ enforce mutual exclusivity between `page`, `offset`, and `size`. Really, `page` and `size` can go together, or `offset` and `size`. Not all 3.

So @damassi , just omit `offset` from the Gravity call (since you are passing in `page` and `size`). Also, it seems like the param has a naming issue, `artworks_by_followed_artists` vs `include_artworks_by_followed_artists`.

Reply:
@ashkan18 / @mzikherman - yup, i'm only concerned with the `mutually exclusive` aspect of the error. Will update!

@mzikherman - this is something I also brought up with Ashkan in a DM; `include_artworks_by_followed_artists` is what is being used in the `filter` endpoint; shouldn't we keep things consistent? Totally cool to update either way, but if we don't additionally update the filter endpoint (and all call-sites) then we'll have to have a branch in our code that says:

```javascript
if (options. include_artworks_by_followed_artists || options. artworks_by_followed_artists) {
  ...
} 
```
With the former pointing to ES and the latter pointing to `/sale_artworks`. 

The idea is to keep what we currently have and seamlessly add in an additional endpoint that requires no changes to existing API.

Reply:
i can easily change new endpoints to `include_artworks_by_followed_artists` but i just feel the name is little misleading, we are not actually _including_ artworks by followed artists, we are _just_ getting them.

Reply:
If the other parameter wasn't already being used in code then I would agree that we should switch it, but at this point it might be best to align with existing code and then loop back and refactor both at once.

Reply:
makes sense, i can switch to include `include` :) and add a proper description on the field describing it's not including anything hehehe PR will come your way soon.